### ISSUE TITLE
Unable to select or drag displays in gnome Display

### ISSUE THREAD

**Issue by galvedro** [linuxmint/cinnamon-control-center#135]
I have a setup consisting of a docked laptop (HP Elitebook) connected to two monitors: an HP L1950 and a FujitsuSiemens. I am running a freshly installed Linux Mint 17.3 Rosa with Cinnamon desktop (2.8.8+rosa).

When I open the Display setup tool to rearrange the display positions, the following happens:

1. The monitors are correctly identified, and their resolutions are correctly initialized.
2. The laptop's screen (inactive in this setup) is also correctly identified.
3. I cannot change the display selection. Whatever is selected, it stays selected no matter where I click.
4. I cannot drag the displays to rearrange their physical layout. Neither the selected display nor the other one.

I have worked around this problem by manually editing the positioning information at ~/.config/monitors.xml

**Comment by PavelNiedoba** [comment]
Similar problem here, linux mint 18, fresh install, can select, but cannot move display to desired positions, using 4 displays, ati open source drivers, the bug is preset for years

**Comment by icarter09** [comment]
@galvedro @PavelNiedoba are either of you still experiencing this issue?

**Comment by PavelNiedoba** [comment]
yes, thought its better now, but there is some "snap helper" which fails to handle 3-4 display configs.
